test/parallel/test-http-write-head.js

```diff
@@ -51,6 +51,12 @@ const s = http.createServer(common.mustCall((req, res) => {
     }
   );
 
+  assert.throws(() => {
+    res.writeHead(200, { meow: 'meow' }, ['invalid', 'headers', 'args']);
```

Comment:
```suggestion
    res.writeHead(200, ['invalid', 'headers', 'args']);
```

or is there a reason for using 3 arguments?

Reply:
Not at all, fixed! Thanks